### ISSUE TITLE
Toolbox could install binaries into separate path

### ISSUE THREAD

**Issue by bfirsh** [docker/toolbox#24]
Toolbox could install the binaries to a separate directory somewhere (`/Library/...`?) and add this to `$PATH` instead of clobbering stuff in `/usr/local/bin`. Don't delete my stuff please!

**Comment by jeffdm** [comment]
Would it make sense to create .bak files, similar to the boot2docker installer?

**Comment by mchiang0610** [comment]
@bfirsh what do you think?

**Comment by bfirsh** [comment]
Yes, .bak files would work. Not as neat as a separate directory, I think, but much better than deleting user data. ;)
